lib/configuration/client_configuration.js

```diff
@@ -0,0 +1,29 @@
+const Levels = Object.freeze({
+  Off: 'OFF',
+  Error: 'ERROR',
+  Warn: 'WARN',
+  Info: 'INFO',
+  Debug: 'DEBUG',
+  Trace: 'TRACE'
+});
+
+const allLevels = Object.values(Levels);
+
+class ClientConfig {
```

Comment:
ClientConfig currently contains only logging config but we don't know how it's going to evelove in the future - let's create nested object called ClientLoggingConfig

Reply:
done